### DISCUSSION TITLE
Pertanyaan untuk mentoring session untuk materi P4DS & PS, 12 Januari 2024, 13.00-15.00

### DISCUSSION THREAD

**Discussion by KZF022329** [teamalgoritma/community#841] [Q&A] [answered]
Tempo hari disampaikan bahwa R dan Python tak berlawanan tapi bersisihan. IMHI (in my humble imagination :D), sepertinya, pengolahan data lebih _adequate_ dilakukan melalui Python. Dalam bayangan saya saat mengikuti R, nantinya data diolah di Python dan divisualisasikan di R (CMIIW). Pertanyaanya:

Bagaimana cara menghubungkan hasil olahan di Python dan R ini?

terima kasih sebelumnya

**Comment by saskia-dwi-ulfah** [comment, marked as answer]
Dear @KZF022329 ,

Agar bisa menghubungkan hasil pengolahan dari Python dan R, terdapat banyak pilihan yang bisa dilakukan, tergantung dengan kebutuhan bisnis dan tools yang digunakan. Salah satu practice yang cukup umum, kita bisa menyimpan hasil pengolahan Python dalam format tertentu, misalnya .csv dan .xlsx dan membuat visualisasi menggunakan package yang terdapat di R.

Di samping itu, saat ini sudah banyak tersedia package yang memungkinkan kita untuk menggunakan Python di dalam R dan sebaliknya. Berikut resource tambahan terkait hal ini:
- [Leveraging the Best of both Python and R](https://www.datacamp.com/tutorial/using-both-python-r).

Semoga jawaban ini dapat membantu.

Best regards,
Team Algoritma
